crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -774,7 +774,7 @@ impl Triangle3d {
         let [a, b, c] = self.vertices;
         let ab = b - a;
         let ac = c - a;
-        ab.cross(ac).length() < 10e-7
+        ab.cross(ac).length() < EPSILON
```

Comment:
So I will just note that at one point the PR that added this used the machine epsilon, and then it was changed to use 10e-7 (they apparently forgot to update the docs). I have a lingering suspicion I suggested this change. @vitorfhc can you fill us in here?

Reply:
Yep, link to that thread here: https://github.com/bevyengine/bevy/pull/12508#discussion_r1534137670

Reply:
Yeah, we shouldn't use machine epsilon here.